sdks/python/apache_beam/transforms/stats_test.py

```diff
@@ -345,5 +353,313 @@ def test_approximate_unique_globally_by_error_with_skewed_data(self):
     pipeline.run()
 
 
+class ApproximateQuantilesTest(unittest.TestCase):
```

Comment:
Is it possible to run a test (even a one off) on Dataflow to see if this is working end to end?

Reply:
I ran an example on dataflow and its working as expected.
https://console.cloud.google.com/dataflow/jobsDetail/locations/us-central1/jobs/2019-08-01_05_54_51-13352737957197087204?project=apache-beam-testing

Reply:
Great!